waspc/cli/Command/Db/Migrate.hs

```diff
@@ -86,3 +94,39 @@ applyAvailableMigrationsAndGenerateClient genProjectRootDir = do
                                genClientError
         Right () -> waspSaysC "Done."
 
+
+data MigrationDirCopyDirection = CopyMigDirUp | CopyMigDirDown deriving (Eq)
+
+-- | Copy migrations directory between Wasp source and the generated project.
+copyDbMigrationsDir
+    :: MigrationDirCopyDirection -- ^ Copy direction (source -> gen or gen-> source)
+    -> Path Abs (Dir WaspProjectDir)
+    -> Path Abs (Dir ProjectRootDir)
+    -> IO (Maybe String)
+copyDbMigrationsDir copyDirection waspProjectDir genProjectRootDir = do
+    let dbMigrationsDirInDbRootDir = SP.fromPathRelDir [P.reldir|migrations|]
```

Comment:
If you are using SP here, you should probably specify in type what it is relative to.